docs/go/topics/client_auth.md

```diff
@@ -0,0 +1,24 @@
+In your server's Go file, we pass a TLS stack configuration into the server initalization. The configuration enables strict client certificate verficiation against all trusted root certificates in the CA pool we create. Finally, we call the server's API to create a HTTPS listener using it's own certificate and private key for client-side verification.
+
+```go
+// ...
+
+caCert, _ := ioutil.ReadFile("{{ ca_cert }}")
+caCertPool := x509.NewCertPool()
+caCertPool.AppendCertsFromPEM(caCert)
+
+tlsConfig := &tls.Config{
+    ClientCAs: caCertPool,
+    ClientAuth: tls.RequireAndVerifyClientCert,
```

Comment:
@sourishkrout A thought here in retrospect. Should you change this to `tls.RequireAnyClientCert` instead, so that further below in a request handler you can show how to check for authorized requests and display a friendly message, instead of just failing the handshake? I'm doing this in both Nginx and Node.js client auth docs.